app/controllers/nonsipp/TaskListController.scala

```diff
@@ -282,19 +290,32 @@ object TaskListController {
     }
   }
 
-  private def memberPaymentsSection(srn: Srn) = {
+  private def memberPaymentsSection(srn: Srn, userAnswers: UserAnswers): TaskListSectionViewModel = {
```

Comment:
I assume this still need to be changed later on, as there are some discussion on Friday, how should this exactly work.

Reply:
Yeah this can be changed, just wanted to add some preliminary logic with the section completed